releasenotes/notes/winnodowngrade-4a630bdd62c9ac17.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+deprecations:
```

Comment:
this should be an `upgrade` note (this change does not really deprecate downgrades, but rather completely disables them)